client/app/assets/styles/base.css

```diff
@@ -1,3 +1,11 @@
+html {
+  box-sizing: border-box;
+  font-size: 14px;
```

Comment:
This is a PR for desktop component, could I live media queries to some other PR?